src/Divider/divider.scss

```diff
@@ -1,8 +1,8 @@
-@import '../utils/theme/variables.scss';
+$color: #E6E6E6;  // pearl
```

Comment:
shouldn't we set all of these variables with a `!default` flag to make them overridable?

e.g.
```scss
$color: #E6E6E6 !default;
```